src/staff/components/StaffProperties/StaffProperties.tsx

```diff
@@ -105,7 +103,7 @@ interface StaffPropertiesProps {
   };
   errors: StaffErrorFragment[];
   disabled: boolean;
-  staffMember: StaffMemberDetailsFragment;
+  staffMember: StaffMemberDetails;
```

Comment:
`StaffMemberDetails` is a mapped type from `StaffMemberDetailsFragment` that makes two fields (isActive, permissionGroups) optional. `StaffMemberDetailsFragment` is the api-related thing, is actually a contract we rely on, why do you change that contract? Could you cover cases as the API indicates?

Reply:
Ok, I changed to types from API. It requires checking if fields exist in objects to prevent type errors, but anyway it might be worth doing it gives the advantage of using API types, not custom ones.